storage/spanstore/interface.go

```diff
@@ -55,6 +55,7 @@ type GetTraceParameters struct {
 	TraceID   model.TraceID
 	StartTime time.Time // optional
 	EndTime   time.Time // optional
+	RawTraces bool      // optional
```

Comment:
this field doesn't make sense in the storage API.

Reply:
@yurishkuro How else should we propagate the `RawTraces` flag through to the query service? Do we want to pass it in as a separate parameter?

Reply:
I was actually thinking about this in the context of [Remote Storage API v2](https://docs.google.com/document/d/1sutIXjVJ_zP13cT9uLWfRH8KCOitPOXEHtgYPUUW06g/edit?tab=t.0). We have a similar issue there - the api_v3 is really the API for the query service. I was hoping we could use it as is for the Remote Storage API v2, but this aspect of additional parameters that make sense to the query service but not to the storage get in the way.

Your problem is easier to solve - we can change query service API not to take `spanstore.GetTraceParameters`, but to take its own struct that extends the storage params with additional fields. Since this is just an internal refactoring, it's easier to pull off (although still messy - lots of tests will probably break, e.g on mocks).

I want to try the same approach for gRPC - the service definitions will have to be different, but the api_v3 args could be extensing storage api args.

Reply:
@yurishkuro Should I open a separate PR to define something like the following in `package querysvc`?
```go
type GetTraceParameters struct {
  spanstore.GetTraceParameters
}
```

Reply:
if it causes a lot of changes then separate PR would make sense

Reply:
https://github.com/jaegertracing/jaeger/pull/6438